djstripe/models/payment_methods.py

```diff
@@ -455,3 +455,60 @@ def detach(self):
 			# https://github.com/stripe/stripe-python/issues/376
 			self.sync_from_stripe_data(self.api_retrieve())
 			return False
+
+
+class PaymentMethod(StripeModel):
+	"""
+	Stripe documentation: https://stripe.com/docs/api#payment_methods
+	"""
+
+	billing_details = JSONField(
+		help_text=(
+			"Billing information associated with the PaymentMethod that may be used or "
+			"required by particular types of payment methods."
+		)
+	)
+	card = JSONField(
+		help_text=(
+			"If this is a card PaymentMethod, this hash contains details about the card."
+		)
+	)
+	card_present = JSONField(
+		help_text=(
+			"If this is an card_present PaymentMethod, this hash contains details about "
+			"the Card Present payment method."
+		)
+	)
+	customer = models.ForeignKey(
+		"Customer",
+		on_delete=models.SET_NULL,
+		null=True,
+		blank=True,
+		related_name="payment_methods",
+		help_text=(
+			"Customer to which this PaymentMethod is saved."
+			"This will not be set when the PaymentMethod has not been saved to a Customer."
+		),
+	)
+	type = models.CharField(
+		max_length=255,
+		null=True,
+		blank=True,
+		help_text=(
+			"The type of the PaymentMethod. An additional hash is included on the PaymentMethod"
+			"with a name matching this value. It contains additional information specific to the"
+			"PaymentMethod type."
+		),
+	)
+
+	stripe_class = stripe.PaymentMethod
+	stripe_dashboard_item_name = "payment methods"
+
+	@classmethod
+	def attach(
+		cls, payment_method_id, stripe_customer, api_key=djstripe_settings.STRIPE_SECRET_KEY
+	):
+		stripe_payment_method = stripe.PaymentMethod.attach(
+			payment_method_id, customer=stripe_customer["id"], api_key=api_key
+		)
+		return cls._create_from_stripe_object(stripe_payment_method)
```

Comment:
I need to test, but I think this should use sync_from_stripe_data instead of _create_from_stripe_object, since the payment_method may already exist in the DB.

Reply:
resolved in ba608b4c9cfee098da6e54cd4658022d74f7fdaa